src/parser/postgresparser.cpp

```diff
@@ -1122,6 +1124,15 @@ parser::DropStatement *PostgresParser::DropTriggerTransform(DropStmt *root) {
   return result;
 }
 
+parser::DropStatement *PostgresParser::DropIndexTransform(DropStmt *root) {
+  auto res = new DropStatement(DropStatement::EntityType::kIndex);
```

Comment:
Please use "result" as the name of the variable that you will be returning for consistency with all other functions within this file.